### PR TITLE
Sample SDK docs

### PR DESCRIPTION
**Pull request by divyaamunugama**
> - React
> - Angular
> - JS

### CODE REVIEW

File: en/docs/sdks/integrate-a-js-app.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Basic OAuth/OpenID Connect configuration
+
+{!fragments/oauth-app-config-basic.md!}
+
+## Download the sample
+Download the [sample JavaScript application](https://github.com/asgardeo/asgardeo-auth-spa-sdk/releases/latest/download/asgardeo-html-js-app.zip).
+
+## Configure the sample
+1. Open the `index.html` file located at the root of the project
```
Comment (nilminiwso2) [inline en/docs/sdks/integrate-a-js-app.md:19]:
```suggestion
1. Open the `index.html` file located at the root of the project.
```

File: en/docs/sdks/integrate-a-js-app.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Integrate a JavaScript app
+Follow the steps given below to authenticate users to a sample JavaScript service provider with OpenID Connect.
+
+## Prerequisites
+- Only customer users can log in to applications. [Create a user account](../../guides/identity-lifecycles/onboard-overview) if you don't already have one.
```
Comment (nilminiwso2) [inline en/docs/sdks/integrate-a-js-app.md:5]: We don't need a bullet point when there is only one point

File: en/docs/sdks/integrate-a-js-app.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Create a service provider
+
+{!fragments/register-a-service-provider.md!}
+
+## Basic OAuth/OpenID Connect configuration
```
Comment (nilminiwso2) [inline en/docs/sdks/integrate-a-js-app.md:11]: this title doesn't match the others. we have to do something about that...

File: en/docs/sdks/integrate-a-js-app.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Integrate a JavaScript app
+Follow the steps given below to authenticate users to a sample JavaScript service provider with OpenID Connect.
+
+## Prerequisites
+- Only customer users can log in to applications. [Create a user account](../../guides/identity-lifecycles/onboard-overview) if you don't already have one.
+
+## Create a service provider
+
+{!fragments/register-a-service-provider.md!}
```
Comment (nilminiwso2) [inline en/docs/sdks/integrate-a-js-app.md:9]: We need a way to incorporate the SP configs relevant to the sample we are using.

**Comment by divyaamunugama** [comment]
Changes with commit [33fa45d](https://github.com/wso2/docs-is/pull/2879/commits/33fa45dcd9a78c3cc7abbd686188f2997a823b47)

1. Added a new folder "Start integrating apps" and added the docs inside it
2. Removed fragments and added content manually to solve this: https://github.com/wso2/docs-is/pull/2879#discussion_r834909518
3. Updated titles where necessary

**Comment by divyaamunugama** [comment]
Changes with commit [1b56e69](https://github.com/wso2/docs-is/pull/2879/commits/1b56e696ef49bd611d944ef2c942423fb5b15754)

Added a new step and a warning to react SDK guide.
Solves issue: https://github.com/wso2/product-is/issues/13363

File: en/docs/sdks/start-integrating-apps/integrate-a-js-app.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# Integrate a JavaScript application
+Follow the steps given below to authenticate users to a sample JavaScript service provider with OpenID Connect.
+
+## Prerequisites
+Only customer users can log in to applications. [Create a user account](../../guides/identity-lifecycles/onboard-overview) if you don't already have one.
+
+## Configure a service provider
+
+1. On the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```
Comment (nilminiwso2) [inline en/docs/sdks/start-integrating-apps/integrate-a-js-app.md:9]:
```suggestion
1. On the WSO2 Identity Server management console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```

File: en/docs/sdks/start-integrating-apps/integrate-a-react-app.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+# Integrate a React application
+Follow the steps given below to authenticate users to a sample react service provider with OpenID Connect.
+
+## Prerequisites
+Only customer users can log in to applications. [Create a user account](../../guides/identity-lifecycles/onboard-overview) if you don't already have one.
+
+## Configure a service provider
+
+1. On the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```
Comment (nilminiwso2) [inline en/docs/sdks/start-integrating-apps/integrate-a-react-app.md:9]:
```suggestion
1. On the WSO2 Identity Server management console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```

File: en/docs/sdks/start-integrating-apps/integrate-an-angular-app.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+# Integrate an Angular application
+Follow the steps given below to authenticate users to a sample angular service provider with OpenID Connect.
+
+## Prerequisites
+Only customer users can log in to applications. [Create a user account](../../guides/identity-lifecycles/onboard-overview) if you don't already have one.
+
+## Configure a service provider
+
+1. On the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```
Comment (nilminiwso2) [inline en/docs/sdks/start-integrating-apps/integrate-an-angular-app.md:9]:
```suggestion
1. On the WSO2 Identity Server management console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**